### PR TITLE
[C-2034] Cancel all download jobs on log out

### PR DESCRIPTION
**Pull request by amendelsohn**
### Description

Cancel all download worker jobs on log out.

Questions:
Should we cancel play count jobs? I think these are okay, but we might record them for the wrong user once we're back online. Maybe need to tag the job with the user id? This is also an edge case so maybe okay to drop them and cancel the jobs.

### CODE REVIEW

File: packages/mobile/src/services/offline-downloader/offline-download-queue.ts [hunk 1]
```diff
@@ -224,3 +224,19 @@ export const cancelQueuedDownloads = async (
...
+  jobs.forEach((rawJob) => {
+    const { workerName, id } = rawJob
+    if (
+      workerName === TRACK_DOWNLOAD_WORKER ||
+      workerName === COLLECTION_DOWNLOAD_WORKER
+    ) {
+      queue.cancelJob(id)
+    }
+  })
+  // TODO: call startIfOffline
```
Comment (amendelsohn) [inline packages/mobile/src/services/offline-downloader/offline-download-queue.ts:240]: should be start if online. Waiting on merge from https://github.com/AudiusProject/audius-client/pull/2773

Comment (nicoback2) [reply packages/mobile/src/services/offline-downloader/offline-download-queue.ts:240]: merged!

**Review by sliptype — APPROVED** [review]
Looks good to me, should we address that todo?

**Comment by amendelsohn** [comment]
> Looks good to me, should we address that todo?

Yep it's on the way

**Pull request merged**